### ISSUE TITLE
ENH: Accurate TBM Filter 

### ISSUE THREAD

**Issue by csmcallister** [GSA/tbm-scan#13]
Given that we have key terms for TBM, when the scanner runs it should *accurately* find TBM solicitations.
  
## Tasks
- [x] Review [scan results](https://docs.google.com/spreadsheets/d/1NSwnX__A2EyDjlZmuWGVDUvS5dXt9qDr4r_AtkM6_gM/edit#gid=950323265&fvid=1263675032) for 10/01/18 - 07/22/19 and identify trends in false positives. Once in the spreadsheet, use the filter view called *Results* to see the solicitations that were found. The current filter methodology is to capture solicitations that contain any of these keywords in any column: 
```python
                {'technology business management',
                 'tbm',
                 'tbma',
                 'it spending transparency',
                 'tbm framework',
                 'it tower',
                 'sub-towers',
                 'cost pool',
                 'sub-pools'}
```
Some of those terms might be responsible for a lot of false positives. Document findings in this issue.
- [ ] Incorporate the above findings into the TBM filter methodology. Re-run scan and then re-evaluate.

**Comment by csmcallister** [comment]
Takeaways for results review:

 -  False positives due to "TBM" being substring (e.g. "L**TBM**U"). Fix by using regular expression with word boundary.
 - False positives also due to looking in irrelevant fields such as ZIP and AWARDEE. Fix by limiting search to the DESC and SUBJECT fields.

**Comment by csmcallister** [comment]
closed by f7f2ad5
